### ISSUE TITLE
[Bug]: Fatal errors in ddtrace.stub.php

### ISSUE THREAD

**Issue by sleepyboy** [DataDog/dd-trace-php#2953]
### Bug report

ddtrace.stub.php causes fatal errors in PHP 8:
```
Fatal error: Type mixed cannot be marked as nullable since mixed already includes null in $PATH/vendor/datadog/dd-trace/ext/ddtrace.stub.php on line 801

PHP Fatal error:  Type mixed cannot be marked as nullable since mixed already includes null in $PATH/vendor/datadog/dd-trace/ext/ddtrace.stub.php on line 811
````

This can be fixed by changing signature from `?mixed $body = null` to `mixed $body = null`.

Once these errors are fixed, there will be multiple `undefined constant` errors:
```Fatal error: Uncaught Error: Undefined constant "DDTrace\UNKNOWN" in $PATH/vendor/datadog/dd-trace/ext/ddtrace.stub.php on line 12

Fatal error: Uncaught Error: Undefined constant "DDTrace\Internal\UNKNOWN" in $PATH/vendor/datadog/dd-trace/ext/ddtrace.stub.php on line 841

Fatal error: Uncaught Error: Undefined constant "UNKNOWN" in $PATH/vendor/datadog/dd-trace/ext/ddtrace.stub.php on line 874
```

These can be fixed by adding something like that to corresponding namespaces:
```
const UNKNOWN = 0;
```

For instance, this would fix the first undefined constant error:
```
<?php
namespace DDTrace {
    const UNKNOWN = 0; // this should fix it

    /**
     * @var int
     * @cvalue DD_TRACE_DBM_PROPAGATION_DISABLED
     */
    const DBM_PROPAGATION_DISABLED = UNKNOWN;
   ....
}
```

How to reproduce:
Create a test PHP file (eg. test.php) and include the stub file:
```
<?php

include('vendor/datadog/dd-trace/ext/ddtrace.stub.php');
```

Execute the file from a command line using PHP 8:
```
php test.php
```

### PHP version

8.3

### Tracer or profiler version

1.4.2

**Comment by bwoebi** [comment]
Hey @sleepyboy,

we just noticed that today and fixed it in #2933, where we removed these extra question marks.

Thanks for reporting though!

**Comment by bwoebi** [comment]
These UNKNOWN constants are special placeholders of the stub generator. We cannot change these or put `UNKNOWN` there, as that would have the stub generator create an actual UNKNOWN constant for dd-trace-php.

I suppose, if you want to actually include the stubs directly, you'll have to define a dummy `const UNKNOWN = 0;` or similar, right before including.
